lib/evaluate_board.rb

```diff
@@ -13,30 +15,36 @@ def initialize(board_gateway)
     ]
   end
 
-  def execute()
+  def execute
```

Comment:
Do you think returning a string is the best option?

Reply:
For now, yes. However, this may or may not change when the AI is introduced.